src/Main.java

```diff
@@ -3,35 +3,87 @@
 
-        // НИЖЧЕ ПРИКЛАД КОДУ, ВИКОРИСТОВУЙТЕ ЙОГО ЯК ЗРАЗОК,
-        // АЛЕ САМ ПРИКЛАД МОЖНА ВИДАЛИТИ ПОТІМ
-        // Приклад сканнера для перехоплення вводу користувача з консолі
         System.out.println("Привіт, відважний герою!");
         System.out.println("Тебе чекає цікава подорож");
+        System.out.println("На старті маєш: здоров'я 100 та удачі 50");
+        System.out.println("Почнемо");
         System.out.println("Ти йшов чарівною стежкою, і натрапив на роздоріжжя: 1 - піти наліво, 2 - піти направо");
-        Scanner sc = new Scanner(System.in);
-        // Таким чином можна отримати цифру яку ввів користувач
-        // і зберегти її в змінну типу int
-        int choice1 = sc.nextInt();
+        System.out.println("Введи будь-яку цифру ");
+
+        int health = 100;
+        int luck = 50;
+        int choice1 = scanner.nextInt();
         if (choice1 == 1) {
             System.out.println("Ти обрав піти наліво");
-            // відняти очки здоров*я
+            health = health - 20;
+            System.out.println("Ти втратив 20 очків здоров'я");
+            System.out.println("Залишок здоров'я " + health);
         } else if (choice1 == 2) {
             System.out.println("Ти обрав піти направо");
-            // додати очки удачі
+            luck = luck + 30;
+            System.out.println("Ти знайшов скарб і отримав 30 очків удачі");
+            System.out.println("Залишок удачі " + luck);
         } else {
             System.out.println("Ти заплутався");
-            // відняти очки здоров*я
+            health = health - 10;
+            System.out.println("Втрачаєш 10 очків здоров'я");
+            System.out.println("Залишок здоров'я " + health);
         }
-        // КІНЕЦЬ ПРИКЛАДУ, ВИДАЛІТЬ ПРИКЛАД, ЩОБ В КОДІ БУЛО ТІЛЬКИ ВАШЕ РІШЕННЯ
-
 
+        System.out.println("Ти натрапив на загадковий колодязь");
+        System.out.println("Введи цифру від 1 до 3-х");
+        health = health;
+        luck = luck;
+        int choice2 = scanner.nextInt();
+        if (choice2 == 1) {
+            System.out.println("Ти п'єш магічну воду і отримуєш 50 очків здоров'я");
+            health = health + 50;
+            System.out.println("Залишок здоров'я " + health);
+        } else if (choice2 == 2) {
+            System.out.println("Ти втомився і втрачаєш 10 очків удачі");
+            luck = luck - 10;
+            System.out.println("Залишок удачі " + luck);
+        } else {
+            System.out.println("Вибір не вплинув на баланс очків");
+            System.out.println("Залишок здоров'я " + health + " та удачі " + luck);
+        }
 
-        // Нижче пишіть ваш код з реалізацією завдання
-        // Коли пишете логіку гри - не забувайте періодично запускати програму
-        // І переконуватись, що все працює як треба, заодно тестуйте її перебираючи інпути
+        System.out.println("Ти зустрів магічну істоту");
+        System.out.println("Введи цифру від 1 до 3-х");
```

Comment:
In the original task this should be **randomized** instead of letting user choose, although I won't say this is a mistake - let's call if "a small game design adjustment"